statsd/statsd.go

```diff
@@ -123,6 +130,13 @@ func New(addr string) (*Client, error) {
 // io.WriteCloser + SetWriteTimeout(time.Duration) error
 func NewWithWriter(w statsdWriter) (*Client, error) {
 	client := &Client{writer: w, SkipErrors: false}
+
+	// Inject DD_ENTITY_ID as a constant tag if found
+	if os.Getenv(entityIDEnvName) != "" {
```

Comment:
I guess storing this in a string should be better than calling `os.Getenv` twice?